modules/migrating-sccs-during-the-manual-upgrade.adoc

```diff
@@ -0,0 +1,360 @@
+// Module included in the following assemblies:
+//
+// * upgrading/upgrade-roxctl.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="migrating-sccs-during-the-manual-upgrade_{context}"]
+= Migrating SCCs during the manual upgrade
+
+By migrating the security context constraints (SCCs) during the manual upgrade by using `roxctl` CLI, you can seamlessly transition the {rh-rhacs-first} services to use the stock {osp} SCCs, ensuring compatibility and optimal security configurations across Central and all secured clusters.
+
+.Procedure
+
+. List all the {product-title-short} services that are deployed on Central and all secured clusters:
++
+[source,terminal]
+----
+$ oc -n stackrox describe pods | grep 'openshift.io/scc\|^Name:'
+----
++
+.Example output
++
+[source,text,subs="+quotes"]
+----
+Name:      admission-control-6f4dcc6b4c-2phwd
+           *openshift.io/scc: stackrox-admission-control*
+#...
+Name:      central-575487bfcb-sjdx8
+           *openshift.io/scc: stackrox-central*
+Name:      central-db-7c7885bb-6bgbd
+           *openshift.io/scc: stackrox-central-db*
+Name:      collector-56nkr
+           *openshift.io/scc: stackrox-collector*
+#...
+Name:      scanner-68fc55b599-f2wm6
+           *openshift.io/scc: stackrox-scanner*
+Name:      scanner-68fc55b599-fztlh
+#...
+Name:      sensor-84545f86b7-xgdwf
+           *openshift.io/scc: stackrox-sensor*
+#...
+----
++
+In this example, you can see that each pod has its own custom SCC, which is specified through the `openshift.io/scc` field.
+
+. Add the required roles and role bindings to use the stock {osp} SCCs instead of the {product-title-short} custom SCCs:
```

Comment:
I'm not clear on what is meant by _stock_ here and in the next step.  I wonder if we can remove it to avoid a potential translation issue?

```suggestion
. Add the required roles and role bindings to use the {osp} SCCs instead of the {product-title-short} custom SCCs:
```

Reply:
We are migrating from SCCs that we built ourselves (stackrox-*) to SCCs that ship with OpenShift (e.g. nonroot-v2), hence I called them "stock" SCCs. Feel free to rephrase this to ensure the instructions are clear!